packages/components/addon/components/hds/form/character-count/index.hbs

```diff
@@ -0,0 +1,19 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+<Hds::Text::Body
+  @tag="div"
```

Comment:
[Question] Should this be a more semantic tag?

Reply:
Fair question – I was replicating what we have for the HelperText and Error, but it could be a `<p>` tag just as well.